### ISSUE TITLE
Rigidbody velocity gets improperly clamped if a max_linear_velocity is set

### ISSUE THREAD

**Issue by Dooskington** [rustsim/nphysics#253]
I have been using `nphysics2d` and notice strange behavior if I set the `max_linear_velocity` on a rigidbody. I am setting the velocity of a ball rigidbody every tick, reflecting that velocity when the body collides into a wall (it's arkanoid/breakout/brickbreaker). I apply a small multiplier as well, so the body speeds up as it hits stuff.

The rigidbody is dynamic, with a mass of 1.0, and gravity disabled. Everything else except the max linear velocity is the default.

When the magnitude of the linear velocity hits the value I set as `max_linear_velocity`, I expect the physics engine to just clamp the body and keep it from going any faster. However, what's happening is the body immediately slows down to a crawl, and the new velocity magnitude is some tiny number. Is this the intended behavior?

My workaround for now is just to clamp the velocity on my end:
```rust
let vel = ball.velocity;
let normal = normal.normalize();
let dot = vel.linear.dot(&normal);

let mut reflected_vel = (vel.linear - (2.0 * dot) * normal) * 1.01;
reflected_vel = reflected_vel.normalize() * nalgebra::clamp(reflected_vel.magnitude(), 0.0, BALL_MAX_LINEAR_VELOCITY);
ball.velocity = Velocity::new(reflected_vel, vel.angular);
```

Here's a low-framerate video of it happening. You can see a ball hitting the top left area, then reflecting and being slowed down. Removing the `max_linear_velocity` call fixes this behavior.

![vm9JILAY7i](https://user-images.githubusercontent.com/4643504/79004951-a8329c80-7b0a-11ea-8474-d8b8bd60c30a.gif)

Will take a look at the code and update this issue if I spot anything of note

**Comment by sebcrozet** [comment]
Hi! No, this is not the intended behavior. There is a silly mistake on the rigid body code where a division and multiplication are reversed. This should be fixed by #254.

**Comment by Dooskington** [comment]
Awesome. Thanks for the quick investigation and fix!
